core/scripts/cli/sponsor/withdraw.js

```diff
@@ -167,16 +161,21 @@ const withdraw = async (web3, artifacts, emp) => {
       .divRound(scalingFactor);
     const excessCollateral = toBN(collateral).sub(minCollateralAboveGcr);
     const maxInstantWithdrawal = excessCollateral.gt(toBN(0)) ? excessCollateral : toBN(0);
-    console.log("Maximum amount you can withdraw instantly:", fromWei(maxInstantWithdrawal));
-    console.log("To withdraw more than this amount, we will issue a withdrawal request first");
+    console.log(
+      "You must request an amount to withdraw. The request takes",
+      withdrawalLivenessInMinutes,
+      "minutes to process."
+    );
+    // TODO: Use price feed to calculate what's the maximum withdrawal amount that'll meet the collateralization
+    // requirement. Display that to the user here.
 
     // Prompt user to enter withdrawal amount
     const input = await inquirer.prompt({
       name: "numCollateral",
       message: "How much " + requiredCollateralSymbol + " to withdraw?",
       validate: value =>
         (value > 0 && toBN(toWei(value)).lte(toBN(collateral))) ||
-        "Number of " + requiredCollateralSymbol + " must be positive and up to your current locked collateral"
+        "You can only withdraw up to " + fromWei(collateral) + " " + requiredCollateralSymbol
```

Comment:
small nit: in some places you join string like this and in others you use `...${}...` notation. We should probably do this in a consistent way.

Reply:
Which way do we prefer? I can create a PR that goes through the whole CLI and adjusts to that way.

Reply:
I've always used the `...${}` way because I don't like string concatenation and I think its easier from the programmer's (i.e. mine) point of view. Do you find the string concatenation easier to write?

Reply:
I'm indifferent. Let me send a PR moving everything to that format.